src/include/migraphx/config.hpp

```diff
@@ -23,12 +23,23 @@
  */
 #ifndef MIGRAPHX_GUARD_CONFIG_HPP
 #define MIGRAPHX_GUARD_CONFIG_HPP
-
+#include <migraphx/version.h>
 namespace migraphx {
 
 #if !defined(MIGRAPHX_USE_CLANG_TIDY) && !defined(DOXYGEN)
+
+#define MIGRAPHX_CONCAT_1(x, y) x##y
+#define MIGRAPHX_CONCAT(x, y) MIGRAPHX_CONCAT_1(x, y)
```

Comment:
Define this as `MIGRAPHX_VERSION_CONCAT` and `MIGRAPHX_VERSION_PRIMITIVE_CONCAT`.

Reply:
Done.